Food_Tracker_App/app/src/main/java/com/example/foodtracker/ui/ingredients/Ingredient.java

```diff
@@ -59,4 +67,12 @@ public String getExpiry() {
     public void setExpiry(String expiry) {
         this.expiry = expiry;
     }
+
+    public boolean isVisible() {
+        return visible;
+    }
+
+    public void setVisible(boolean visible) {
+        this.visible = visible;
+    }
```

Comment:
We should make the visible attribute private and then have a single method to toggle visibility!

Reply:
Changed it so I didn't even need the isVisible and setVisible in the ingredient class (we can just use a boolean and if statement to change visibility of textview items)